test/index.test.js

```diff
@@ -1,66 +1,118 @@
 'use strict'
 
-const chai = require('chai').use(require('dirty-chai'))
-const { expect } = chai
+const mockRestConf = jest.fn()
```

Comment:
usually we use mocha as standard lib for tests and not jest, please restore mocha and use https://www.npmjs.com/package/bfx-api-mock-srv as mock lib for restv2 test

Reply:
hmm, i see, alright

Reply:
@vigan-abd there is no `pub:map:currency:sym` method support [here](https://github.com/bitfinexcom/bfx-api-mock-srv/blob/master/lib/rest2/methods.js#L10). Should I add it to the lib or should I just mock the call in place?

Reply:
as discussed, there is 1 more thing: the mock api works on the localhost while the lib uses pub bitfinex api and there is no possibility to switch api url